src/main/webapp/dataverse.xhtml

```diff
@@ -61,14 +61,14 @@
                         <!-- Edit Info Panel -->
                         <div class="row" jsf:rendered="#{DataversePage.ownerId != null}">
                             <div class="col-md-6 form-group">
-                                <h:outputLabel for="#{DataversePage.editMode == 'CREATE' ? 'selectHostDataverse' : 'hostDataverseStatic'}" styleClass="control-label">
```

Comment:
@mheppler out of curiosity, what did this h-> p change?
(the one below changes the "for" to an "id" - should that have happened here?)

Reply:
PrimeFaces form inputs have the crazy dynamic ID hierarchy (e.g. `dataverseForm:selectHostDataverse`) and you need a special PrimeFaces outputLabel component as compared to the basic JSF `h:` version in order for the labels and input components to be able to reference one another using their ID's. The input is properly reference in this basic JSF outputLabel, but it can't find the special PrimeFaces autoComplete component in the form.